extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -61,6 +61,37 @@ static public function newFromId( $id, $master = false ) {
 		return null;
 	}
 
+	/**
+	 * @param array $ids
+	 * @return array
+	 */
+	static public function newFromIds( $ids ) {
+		wfProfileIn( __METHOD__ );
+
+		$titles = Title::newFromIDs( $ids );
+		$wallMessages = [ ];
+		$correctIds = [ ];
+
+		//double check if all titles are correct
+		foreach ( $titles as $title ) {
+			if ( $title instanceof Title && $title->exists() ) {
```

Comment:
I will leave this place without log, and remove instance of check. In this place we should never have empty title.